WordPress/src/main/java/org/wordpress/android/ui/sitecreation/segments/NewSiteCreationSegmentsViewModel.kt

```diff
@@ -101,10 +101,10 @@ class NewSiteCreationSegmentsViewModel
     }
 
     data class UiState(
```

Comment:
However you prefer is fine by me, I don't see one being clearly better than other here. I do personally prefer the `companion` default approach right now, because I like to ensure initialization is specific, but here it really doesn't matter.